### DISCUSSION TITLE
General discussion and polls

### DISCUSSION THREAD

**Discussion by pratyakshm** [pratyakshm/WinRice#9] [General]
👋 Welcome!
  I've started using Discussions for people who use, want to contribute to, or just talk about WinRice or related technology/software.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Welcome others and are open-minded. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SakthivelNadar** [comment]
sir can it debloat windows 7?

**Reply by pratyakshm** [reply]
> sir can it debloat windows 7?

oh no

**Reply by SakthivelNadar** [reply]
oh yes sir

**Comment by FireHound-lab** [comment]
Can we debloat a whole ISO file by using this ??

**Reply by pratyakshm** [reply]
You can perform a clean installation of the ISO file inside a VM, perform your changes using WinRice, then use Sysprep to generalize your Windows installation, post which you'll need capture the WIM file of your installation.
Learn more: 
- [How to Create Windows 10 ISO image from existing installation on TenForums](https://www.tenforums.com/tutorials/72031-create-windows-10-iso-image-existing-installation.html) 
- [Sysprep documentation on Microsoft Docs](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation)

**Comment by KenHV** [comment]
Thanks for this praty pro sar! You made my Windows experience bearable 😄

**Reply by pratyakshm** [reply]
> 
> 
> Thanks for this praty pro sar! You made my Windows experience bearable 😄

Good to know that. :)

**Comment by pratyakshm** [comment]
In the last few Windows 10 Insider Dev channel flights, the new open source [Windows Terminal](https://github.com/microsoft/terminal) has been included as an inbox app - and this decision will most likely be persistent by Microsoft.   
Personally I think Windows Terminal is a great app, and I always use it during development work. Hence, I decided not to remove that app in CleanWin v0.3.6. But since CleanWin is a community powered project, user feedback matters the most.
Thus, I have decided to run a poll for the same.  
Please voice your opinion through this poll, your feedback will be received and acted upon.
I have decided to keep this poll open until 30th April, 2021.
[![](https://api.gh-polls.com/poll/01F2V8N0BHQTA9B6GDA6DJBDW2/Yes%2C%20remove%20Windows%20Terminal)](https://api.gh-polls.com/poll/01F2V8N0BHQTA9B6GDA6DJBDW2/Yes%2C%20remove%20Windows%20Terminal/vote)
[![](https://api.gh-polls.com/poll/01F2V8N0BHQTA9B6GDA6DJBDW2/No%2C%20keep%20Windows%20Terminal)](https://api.gh-polls.com/poll/01F2V8N0BHQTA9B6GDA6DJBDW2/No%2C%20keep%20Windows%20Terminal/vote)

My decision on this will be the final on the matter. Thanks and have a good day everyone.
